app/assets/stylesheets/components.scss

```diff
@@ -16,26 +16,19 @@
 }
 
 // Home ==================================================================
-.dgu-home-search {
-  padding-top: 2.5em;
-}
-
 .dgu-topics {
-
-  margin-top: 2em;
-
-  h2 {
-    font-weight: bold;
+  &__heading {
+    margin-bottom: 5px;
   }
+  &__list {
+    columns: 3;
 
-  @media (min-width: 641px) {
     li {
-      min-height: 6em;
+      display: inline-block;
```

Comment:
To help with browsers that don't support columns, could we add a width?
```suggestion
      display: inline-block;
      width: 100%;
```

Before: 
![image](https://user-images.githubusercontent.com/1732331/88161334-c62e9380-cc07-11ea-8d16-cc2d473e0c29.png)

After:
![image](https://user-images.githubusercontent.com/1732331/88161412-dcd4ea80-cc07-11ea-9827-749c2001b25f.png)